### ISSUE TITLE
Issue with pdf.js viewer vset commands in all installer scripts

### ISSUE THREAD

**Issue by g7morris** [Islandora-Collaboration-Group/ISLE#194]
Per my testing comment in the [ISLE 1.1 RC Testing Matrix](https://docs.google.com/spreadsheets/d/1L-wrivXq2pUz7vcGsMCx3X7yKf27uokoaR8SovU_BsU) and from the overall testing process I think we should turn on the `pdf.js` viewer by default for PDFs. 

Otherwise new folks could think the viewer is broken or not acting properly and start filing tickets. Given one cannot scroll through the pdf etc with no viewer it is a super pain by default. 

* Without `pdf.js` viewer on
![screen shot 2018-11-14 at 15 20 15](https://user-images.githubusercontent.com/501554/48510196-1a715580-e821-11e8-8406-a55b60caa8cc.png)

* With `pdf.js` viewer on
![screen shot 2018-11-14 at 15 19 31](https://user-images.githubusercontent.com/501554/48510159-0168a480-e821-11e8-914b-4ab46830803b.png)

@br2490 You appeared to agree in the back and forth in the comments of the ISLE 1.1 RC Testing Matrix. Whaddya think, I'm sure we can figure out the vset for this in the install script, no?

**Comment by br2490** [comment]
Is this already implemented for new installs? https://github.com/Islandora-Collaboration-Group/ISLE-Drupal-Build-Tools/blob/master/isle_islandora_installer.sh#L62

**Comment by g7morris** [comment]
^ Ah it looks like it might be failing then. I ran the [migration site vset script](https://github.com/Islandora-Collaboration-Group/ISLE-Drupal-Build-Tools/blob/master/migration_site_vsets.sh) at least twice during a new site setup and migration with no positive results. Can there be further testing to confirm or deny?

**Comment by br2490** [comment]
Yes. That migration script hasn't been touched since inception. If you have others to report please let me know

**Comment by br2490** [comment]
@g7morris could you do a drush vget format=json and grep for the appropriate PDF settings to incorporate?

**Comment by g7morris** [comment]
@br2490 I'm not in a position to test this further in a timely manner and sadly that testing environment was destroyed. Any chance that someone in the community can pick this testing thread up?

**Comment by DiegoPino** [comment]
@g7morris @br2490 issue there is that pdfjs has no viewer hook, so viewer is defined as part of the islandora_pdf module.

See:

```Shell
drush -u 1 -y vget --format=json islandora_pdf
0
1
0
"\/usr\/bin\/gs"
"\/usr\/bin\/pdftotext"
"700"
"500"
"200"
"200"
{"name":{"none":"none","islandora_pdfjs":"islandora_pdfjs"},"default":"islandora_pdfjs"}
````

Best practice there would be to fix the module so islandora_pdfj becomes part of the "viewers" menu/ hook, quick fix is to set the whole settings, which also kinda makes sense in general?

Up to you folks, can contribute any of those.

Cheers

**Comment by g7morris** [comment]
Is it possible that the issue is the spaces within the vset?  I can't imagine why this would be related or the relative impact (_if any_)?

My code editor seems to not like the `islandora_pdf_viewers` line in particular. Angry red. 

Reviewing all of the other vsets it appears there are no spaces?

Compare: 
            
`'{"name": {"none": "none","islandora_pdfjs": "islandora_pdfjs"},"default": "islandora_pdfjs"}'
'{"name":`
 
to 
                          
`{"none":"none","islandora_openseadragon":"islandora_openseadragon"},"default":"islandora_openseadragon"}'

**Comment by fedunham** [comment]
+1 for inclusion in upcoming release.

**Comment by g7morris** [comment]
Closing as "fixed" from ISLE 1.5.5 release
